tests/integration/modules/molecule/asset/molecule.yml

```diff
@@ -2,6 +2,15 @@
 scenario:
   name: asset
 platforms:
-  - name: instance
-    image: sensu/sensu:latest
-    command: sensu-backend start --state-dir /var/lib/sensu/sensu-backend/etcd1 --state-dir /var/lib/sensu/sensu-backend/etcd1 --log-level debug
+  - name: v5.13.1
```

Comment:
Wouldn't we want to test against `latest`, too? Doing so our daily tests would "detect" incompatible version the same day it's released by Sensu. Currently we would need to manually update the version here to see if the new version works as expected.

Reply:
I was thinking a bit about this, and testing against nightly would make sense since we would be able to catch regressions before release. But as you said, testing against the latest release should be better than nothing.